pubspec.yaml

```diff
@@ -39,6 +39,7 @@ dependencies:
   flutter:
     sdk: flutter
   vibration: ^1.8.1
+  shared_preferences: ^2.2.1
```

Comment:
We're already using secure_storage_provider, maybe it's better to use that instead of sharedPrefs? Not sure what the tradeoffs would be, but better to look into it.